Documentation/gettingstarted/kubeproxy-free.rst

```diff
@@ -1267,6 +1267,22 @@ As per `k8s Service <https://kubernetes.io/docs/concepts/services-networking/ser
 Cilium's eBPF kube-proxy replacement by default disallows access to a ClusterIP service from outside the cluster.
 This can be allowed by setting ``bpf.lbExternalClusterIP=true``.
 
+VLAN 802.1q support
+*******************
+
+Cilium enables firewall on used native devices and will filter all unknown traffic. VLAN 802.1q packets
+will be always passed through their main device with tag (e.g. vlan device is eth0.4000 and it's main interface is eth0).
```

Comment:
```suggestion
will always be passed through their main device and associated tag (e.g. VLAN device is ``eth0.4000`` and its main interface is ``eth0``).
```

Reply:
Adding formatting to `eth0` should also fix the spell check step in the failing GitHub Action.

Reply:
Resolved, but I've changed 'and' to 'with', cause packets are tagged (with tag) and passed through main device... Sorry, english is not my native language, but somebody had to write something :)